dynatrace-aws-logs.sh

```diff
@@ -18,8 +18,8 @@ readonly LAMBDA_ZIP_NAME="dynatrace-aws-log-forwarder-lambda.zip"
 
 readonly DEFAULT_STACK_NAME="dynatrace-aws-logs"
 
-readonly DYNATRACE_TARGET_URL_REGEX="^https:\/\/[-a-zA-Z0-9@:%._+~=]{1,256}$"
-readonly ACTIVE_GATE_TARGET_URL_REGEX="^https:\/\/[-a-zA-Z0-9@:%._+~=]{1,256}\/e\/[a-z0-9-]{1,36}$"
+readonly DYNATRACE_TARGET_URL_REGEX="^https:\/\/[-a-zA-Z0-9@:%._+~=]{1,256}[\/]{0,1}$"
+readonly ACTIVE_GATE_TARGET_URL_REGEX="^https:\/\/[-a-zA-Z0-9@:%._+~=]{1,256}\/e\/[-a-z0-9]{1,36}[\/]{0,1}$"
```

Comment:
`If you want to have the new ActiveGate created during deployment, set it to your Dynatrace SaaS environment: https://<your_environment_ID>.live.dynatrace.com`
DYNATRACE_TARGET_URL_REGEX="^https:\/\/**[-a-zA-Z0-9@:%._+~=]{1,256}**[\/]{0,1}$"
> `[-a-zA-Z0-9@:%._+~=]{1,256}`

`If you want to use an existing ActiveGate, set it to your ActiveGate endpoint: https://<your_activegate_IP_or_hostname>:9999/e/<your_environment_ID>`
ACTIVE_GATE_TARGET_URL_REGEX="^https:\/\/[-a-zA-Z0-9@:%._+~=]{1,256}\/e\/**[-a-z0-9]{1,36}**[\/]{0,1}$"
> `[-a-z0-9]{1,36}`

aren't both of those same env/tenantId section with different regex patterns? first probably also has full dnsname, so just asking

Reply:
[\/]{0,1}
=
[\/]?

https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html
(not the most apt reference for bash regex, but I do think this is universal)